src/ensembl/src/content/app/custom-download/containers/content/preview-download/PreviewDownload.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { RootState } from 'src/store';
+import { Link } from 'react-router-dom';
+import { getAttributes } from '../attributes-accordion/state/attributesAccordionSelector';
+import { getFilters } from '../filter-accordion/state/filterAccordionSelector';
+
+import { ReactComponent as closeIcon } from 'static/img/track-panel/close.svg';
+import styles from './PreviewDownload.scss';
+
+import { getSelectedAttributes } from '../result-holder/helpers';
+
+import {
+  setShowPreview,
+  toggleTabButton
+} from '../../../state/customDownloadActions';
+
+import ImageButton from 'src/shared/image-button/ImageButton';
+
+import { CustomDownloadAttributes } from 'src/content/app/custom-download/types/Attributes';
+
+type StateProps = {
+  attributes: CustomDownloadAttributes;
+  filters: {};
+};
+
+type DispatchProps = {
+  setShowPreview: (setShowPreview: boolean) => void;
+  toggleTabButton: (toggleTabButton: string) => void;
+};
+
+type Props = StateProps & DispatchProps;
+
+const PreviewDownload = (props: Props) => {
+  const changeView = (tab: string) => {
+    props.setShowPreview(false);
+    props.toggleTabButton(tab);
+  };
+
+  const attributesList: [] = getSelectedAttributes(props.attributes);
+
+  return (
+    <div className={styles.previewDownload}>
+      <span className={styles.closeButton}>
+        <ImageButton
+          description={'Close preview'}
+          image={closeIcon}
+          onClick={() => {
+            changeView('attributes');
+          }}
+        />
+      </span>
+      <table className={styles.previewDownloadTable}>
+        <tbody>
+          <tr className={styles.previewDownloadHeader}>
+            <td>Species</td>
+            <td>Attributes</td>
+            <td>Filters</td>
+          </tr>
+          <tr>
+            <td>
+              <table>
```

Comment:
Can’t check what this visually corresponds to at the moment, but a `table` tag within a `table` tag looks suspicious. Unless it’s really, semantically, supposed to be a table within a table, can’t we use a more modern styling technique than table layout?

Reply:
Replaced the inner table with Div tags.